### PR TITLE
Cleandetailed

### PR DESCRIPTION
**Pull request by gork3n**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)

### Description

.PhysicalPercentUsed verbiage fix for CPU to MEM (or RAM)

Fix:
Removed CPU block and .PhysicalPercentUsed to MEM block

Result:
image

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/clean-detailed.omp.json [hunk 1]
```diff
@@ -33,15 +33,14 @@
           "foreground": "#ffffff",
           "leading_diamond": "\ue0b2",
           "style": "diamond",
-          "template": "\uf85a CPU: {{ round .PhysicalPercentUsed .Precision }}% | ",
+          "template": "\uf85a MEM: {{ round .PhysicalPercentUsed .Precision }}% | {{ (div ((sub .PhysicalTotalMemory .PhysicalFreeMemory)|float64) 1000000000.0) }}/{{ (div .PhysicalTotalMemory 1000000000.0) }}GB \uf85a ",
+          "trailing_diamond": "<transparent,#516BEB>\ue0b2</>",
           "type": "sysinfo"
         },
         {
           "background": "#516BEB",
```
Comment (JanDeDobbeleer) [inline themes/clean-detailed.omp.json]: this entire block should be removed, right?

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/clean-detailed.omp.json (hunk 1 above)
Comment (gork3n) [reply themes/clean-detailed.omp.json]: Yes, the entire CPU block

File: themes/clean-detailed.omp.json [hunk 2]
```diff
@@ -33,15 +33,14 @@
           "foreground": "#ffffff",
           "leading_diamond": "\ue0b2",
           "style": "diamond",
-          "template": "\uf85a CPU: {{ round .PhysicalPercentUsed .Precision }}% | ",
```
Comment (gork3n) [inline themes/clean-detailed.omp.json:36]: Copied `{{ round .PhysicalPercentUsed .Precision }}% |` in front of MEM: and deleted this line

File: themes/clean-detailed.omp.json [hunk 3]
```diff
@@ -33,15 +33,14 @@
...
+          "template": "\uf85a MEM: {{ round .PhysicalPercentUsed .Precision }}% | {{ (div ((sub .PhysicalTotalMemory .PhysicalFreeMemory)|float64) 1000000000.0) }}/{{ (div .PhysicalTotalMemory 1000000000.0) }}GB \uf85a ",
```
Comment (gork3n) [inline themes/clean-detailed.omp.json:36]: I copied this line in from the block below (deleted in this commit) and added `{{ round .PhysicalPercentUsed .Precision }}% |` from the deleted line 36 above.

File: themes/clean-detailed.omp.json [hunk 4]
```diff
@@ -33,15 +33,14 @@
...
+          "trailing_diamond": "<transparent,#516BEB>\ue0b2</>",
```
Comment (gork3n) [inline themes/clean-detailed.omp.json:37]: Both this line and line 36 were added to this block from the block deleted below.

File: themes/clean-detailed.omp.json (hunk 1 above)
Comment (gork3n) [reply themes/clean-detailed.omp.json]: This was actually the original `MEM:` block. Sorry for the confusion.

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @gork3n for design

**Pull request merged**